keepercommander/commands/audit_alerts.py

```diff
@@ -843,6 +855,138 @@ def execute(self, params, **kwargs):
         AuditSettingMixin.invalidate_alerts()
         logging.info('Alert counts reset to zero')
 
+class AuditAlertEnable(EnterpriseCommand, AuditSettingMixin):
+    def get_parser(self):
+        return alert_enable_parser
+
+    def execute(self, params, **kwargs):
+        enable_all = kwargs.get('enable_all', False)
+        target = kwargs.get('target')
+        
+        if enable_all:
+            if target:
+                raise CommandError('Cannot specify both alert target and --all flag')
+            self._enable_all_alerts(params)
+        else:
+            if not target:
+                raise CommandError('Alert target is required unless using --all flag')
+            self._enable_single_alert(params, target, kwargs)
```

Comment:
This conditional can be simplified here and in disabled_all case too.

Something like this can be used to make code understandable. Please check if my suggestion covers all your cases and feel free to change if you feel like it.
```suggestion
        if enable_all and target:
            raise CommandError('Cannot specify both alert target and --all flag')
        elif enable_all:
            self._enable_all_alerts(params)
        elif not target:
            raise CommandError('Alert target is required unless using --all flag')
        else:
            self._enable_single_alert(params, target, kwargs)
```

Reply:
Okay